src/server/rendering/TerminalRenderingController.cpp

```diff
@@ -1,6 +1,6 @@
 #include "TerminalRenderingController.hpp"
 
-TerminalRenderingController::TerminalRenderingController() : m_termboxTextOutputArea(10) {
+TerminalRenderingController::TerminalRenderingController(uint32_t tcp_port) : m_termboxTextOutputArea(10) {
```

Comment:
Why are you passing this as a constructor instead of setting it like the other values via draw?

Reply:
no real reason, doing it this way meant not saving a variable in ServerGameController. less code/noise